transport/grpc/integration_test.go

```diff
@@ -120,11 +120,36 @@ func TestGRPCNamedErrorNoMessage(t *testing.T) {
 	t.Parallel()
 	doWithTestEnv(t, nil, nil, nil, func(t *testing.T, e *testEnv) {
 		e.KeyValueYARPCServer.SetNextError(yarpcerrors.Newf(yarpcerrors.CodeUnknown, "").WithName("bar"))
-		_, err := e.GetValueGRPC(context.Background(), "foo")
+		err := e.SetValueGRPC(context.Background(), "foo", "bar")
 		assert.Equal(t, status.Error(codes.Unknown, "bar"), err)
 	})
 }
 
+func TestYARPCResponseAndError(t *testing.T) {
+	t.Parallel()
+	doWithTestEnv(t, nil, nil, nil, func(t *testing.T, e *testEnv) {
+		err := e.SetValueYARPC(context.Background(), "foo", "bar")
+		assert.NoError(t, err)
+		e.KeyValueYARPCServer.SetNextError(status.Error(codes.FailedPrecondition, "bar 1"))
+		value, err := e.GetValueYARPC(context.Background(), "foo")
+		assert.Equal(t, "bar", value)
+		assert.Equal(t, yarpcerrors.Newf(yarpcerrors.CodeFailedPrecondition, "bar 1"), err)
+	})
+}
+
+func TestGRPCResponseAndError(t *testing.T) {
+	t.Skip()
```

Comment:
I could just delete this test - this is because the current generated grpc-go clients do not support this behavior. I'm working on changing this.